libretroshare/src/util/rsnet.cc

```diff
@@ -174,3 +497,18 @@ std::string rs_inet_ntoa(struct in_addr in)
 	rs_sprintf(str, "%u.%u.%u.%u", (int) bytes[0], (int) bytes[1], (int) bytes[2], (int) bytes[3]);
 	return str;
 }
+
+std::string rs_inet_ntoa(const in6_addr& in)
```

Comment:
instead of re-implementing this, i think you can use `sockaddr_storage_tostring` declared in `rsnet.h` we should also have already in that files utilities functions to convert back and forth between `in6_addr` and `sockaddr_storage`